src/celengine/render.cpp

```diff
@@ -140,6 +140,8 @@ static const float MaxAsterismLinesConstDist   = 600.0f;
 static const float MaxAsterismLabelsDist = 20.0f;
 static const float MaxAsterismLinesDist  = 6.52e4f;
 
+static const float MaxPointSize  = 64.0f;
```

Comment:
don't hardcode this. use
```
GLint pointSizeRange[2];
glGetIntegerv(GL_SMOOTH_POINT_SIZE_RANGE, pointSizeRange);
MaxPointSize = pointSizeRange[1];
```
somewhere in `Renderer::Renderer()`

Reply:
maybe we can put it in gl::init?

Reply:
and should we use GL_ALIASED_POINT_SIZE_RANGE for gles?

Reply:
> maybe we can put it in gl::init?

sounds like a good idea



> and should we use GL_ALIASED_POINT_SIZE_RANGE for gles?

yes

Reply:
done